dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py

```diff
@@ -42,5 +42,38 @@ def lease(db, dhcp_interface):
     click.echo(tabulate(table, headers=headers))
 
 
+def count_ipv4(start, end):
+    ip1 = int(ipaddress.IPv4Address(start))
+    ip2 = int(ipaddress.IPv4Address(end))
+    return ip2 - ip1 + 1
+
+
+@ipv4.command()
+@click.argument('range_name', required=False)
+@clicommon.pass_db
+def range(db, range_name):
```

Comment:
Suggest to constraint invoking of dhcp_server related cli when dhcp_server feature is not enabled (In config_db, FEATURE["dhcp_server"]["state"] == "enabled")